install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,122 +11,359 @@
-unique names for persistent volumes, but the uniqueness of the NFS volume's
-server and path is up to the administrator.
+Disk partitions can be used to enforce disk quotas and size constraints. Each partition
+can be its own export. Each export is one persistent volume (PV). Openshift enforces
+unique names for PVs, but the uniqueness of the NFS volume's server and path is up to
+the administrator.
 
-Enforcing quotas in this way allows the end user to request persistent storage
+Enforcing quotas in this way allows the developer to request persistent storage
 by a specific amount (e.g, 10Gi) and be matched with a corresponding volume of
 equal or greater capacity.
 
 [[nfs-volume-security]]
-
-=== Volume Security
-Users request storage with a `*PersistentVolumeClaim*`. This claim only lives in
-the user's namespace and can only be referenced by a pod within that same
-namespace. Any attempt to access a persistent volume across a namespace causes
-the pod to fail.
-
-Each NFS volume must be mountable by all nodes in the cluster.
+== Volume Security
+This section covers aspects of Security Context Constraints (SCCs), which are described
+elsewhere, see
+link:../../admin_guide/manage_scc.html[managing SCCs] and
+link:../../architecture/additional_concepts/authorization.html#security-context-constraints[SCC concepts].
+The reader is expected to understand the basics of POSIX permissions, process UIDs,
+supplemental groups, and selinux.
+
+*NOTE:* The
+link:pod_security_context.html[pod and storage security] document should be read before
+implementing NFS volumes.
+
+Developers request NFS storage by referencing, in the `volumes` section of their
+pod spec, either a _PersistentVolumeClaim_ by name, or the NFS volume plug-in directly.
+
+The *_/etc/exports_* file, on the NFS server, contains the accessible NFS directories.
+The target NFS directory has the familiar POSIX owner and group ids. The Openshift NFS
+plug-in mounts the container's NFS directory with the same POSIX ownership and permissions
+found on the exported NFS directory. However, the container is not run with its effective
+UID equal to the owner of the NFS mount, which is the desired behavior.
+
+As an example, if the target NFS directory appears as:
+[[nfs-export]]
+```
+#on the nfs server:
+# ls -lZ /opt/nfs -d
+drwxrws---. nobody 5555 unconfined_u:object_r:usr_t:s0   /opt/nfs
+
+# id nobody
+uid=99(nobody) gid=99(nobody) groups=99(nobody)
+```
+then the container will need to match selinux labels, and either run with a UID of 99
+(_nobody_ owner), or with 5555 in its supplemental groups, in order to access the directory.
+
+=== User IDs
+User ids can be defined in the container image or in the pod spec. The
+link:pod_security_context.html#user-id[pod and storage security] document covers
+controlling storage access based on user ids.
+
+The container needs it's UID set to 99 (ignoring group ids for the moment), so the
+following fragement can be added to the pod spec:
+```
```

Comment:
@erinboyd I see many references to "nobody" in various docs. What are we advocating instead?